aws_cloudtrail_rules/aws_iam_user_recon_denied.py

```diff
@@ -20,6 +20,11 @@ def rule(event):
     if deep_get(event, "userIdentity", "type") != "IAMUser":
         return False
 
+    # Console Activity can easily result in false positives as some pages contain a mix of
+    # items that a user may or may not have access to.
+    if event.get("userAgent").startswith("aws-internal/3"):
```

Comment:
I'm guessing the answer is no, but is there any field besides `userAgent` we can rely on for this? Perhaps `userIdentity.invokedBy` or maybe a list of AWS owned IP addresses that are just used for console actions (unlikely, I know as they're probably shared by all lambdas/EC2s).

Seeing `userAgent` based exceptions just makes me sad because they're so easy to spoof.